### PR TITLE
SAK-33912 - Update conversion script for changed column sizes

### PR DESCRIPTION
**Pull request by jonespm**
Includes fixes fo
-- SAK-32101 - Assignment service refactor
-- KNL-945 Hibernate changes$
-- KNL-1484 Reduce column lengths to support mysql utf8mb4 databases
-- KNL-1566
-- SAK-33406 - Allow reorder of LTI plugin tools
-- SAK-33896 Remove this site association table

### CODE REVIEW

File: docs/conversion/sakai_12_mysql_conversion.sql [hunk 1]
```diff
@@ -769,3 +771,31 @@ CREATE TABLE ASN_SUBMISSION_SUBMITTER (
...
+
+-- SAK-33896 Remove this site association table
+drop table IF EXISTS SITEASSOC_CONTEXT_ASSOCIATION;
+-- END SAK-33896
+
+-- KNL-945 Hibernate changes
+ALTER TABLE CHAT2_MESSAGE
+    DROP FOREIGN KEY FK720F9882555E0B79;
+    MODIFY CHANNEL_ID varchar(36)
+    MODIFY MESSAGE_ID varchar(36);
```
Comment (ottenhoff) [inline docs/conversion/sakai_12_mysql_conversion.sql]: this looks wrong ....

Comment (jonespm) [reply docs/conversion/sakai_12_mysql_conversion.sql]: Well, I'm assuming that all FK's are named the same and that it's generated based on some naming algorithm If this isn't the case it's either going to have to.

- Figure out what named FK to drop (I had an idea for this but it wasn't working right so I need a better idea)
- Change the size in the code back up to 99
- Just not bother changing the size from 99 to 36

Suggestions accepted!

Comment (ottenhoff) [reply docs/conversion/sakai_12_mysql_conversion.sql]: you have semicolon ending line 784 .... meant to be a comma?

Comment (jonespm) [reply docs/conversion/sakai_12_mysql_conversion.sql]: Well that part is certainly wrong! :) What do you think about the FK name? I was hoping it's okay, maybe I should check a few Monday.

File: docs/conversion/sakai_12_mysql_conversion.sql [hunk 2]
```diff
@@ -769,3 +771,31 @@ CREATE TABLE ASN_SUBMISSION_SUBMITTER (
...
+-- END SAK-32101
+
+-- SAK-33896 Remove this site association table
+drop table IF EXISTS SITEASSOC_CONTEXT_ASSOCIATION;
+-- END SAK-33896
+
+-- KNL-945 Hibernate changes
+ALTER TABLE CHAT2_MESSAGE
+    DROP FOREIGN KEY FK720F9882555E0B79
+    MODIFY CHANNEL_ID varchar(36)
```
Comment (ottenhoff) [inline docs/conversion/sakai_12_mysql_conversion.sql]: need a comma after this statement

**Pull request merged**